podman/domain/system.py

```diff
@@ -44,6 +44,10 @@ def login(
         registry: Optional[str] = None,
         reauth: Optional[bool] = False,  # pylint: disable=unused-argument
         dockercfg_path: Optional[str] = None,  # pylint: disable=unused-argument
+        auth: Optional[str] = None,
+        identitytoken: Optional[str] = None,
+        registrytoken: Optional[str] = None,
+        tls_verify: Optional[Union[bool, str]] = None,
```

Comment:
If you agreed the usage of `tls_verify` and let me know how it should be handled exactly, I will change it if it's needed.

Reply:
I will track it and discuss it async, it's a good thing to have and I would make it more general on a future PR. but for now let's move on with this change :)

Reply:
Not necessarily related to this discussion, but since this would require adding `Union` to be imported, how about importing `from __future__ import annotations` (required for Python<3.10 in this context) and doing  `Optional[bool | str]` ?  
 i.e. not changing those `Optional[str]` to `str | None`, but since there is extra import...

..oh wait, this project still supports Python <3.8, so it's a no-go. Anyway, leaving the comment.